manifests/gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -284,6 +286,10 @@ spec:
           - name: SDS_ENABLED
             value: "{{ .Values.global.sds.enabled }}"
           volumeMounts:
+{{- if eq .Values.global.pilotCertProvider "citadel" }}
+          - mountPath: /etc/istio/citadel-ca-cert
```

Comment:
Can you explain why we want to mount the cert? Isn't Pilot using the cert generated by Citadel running inside it?

Reply:
Pilot uses the cert generated by Citadel. The mouting of the cert is for the TLS between data plane and control plane.